bin/gui_test.in

```diff
@@ -16,77 +16,77 @@ ERT_SHARE_PATH = "%s/share" % ERT_ROOT
 ert = os.path.join( ERT_ROOT , "bin/ert" )
 
 def updateEnvVar(var, value):
-    print "|  * %s=%s" % (var,value)
+    print("|  * %s=%s" % (var,value))
     os.environ[var] = value
 
 
 def updateEnvPath(path_var, value):
     if path_var in os.environ:
         path = "%s%s%s" % (value, os.pathsep, os.environ[path_var])
-        print "|  * %s=%s:$%s" % (path_var , value , path_var)
+        print("|  * %s=%s:$%s" % (path_var , value , path_var))
```

Comment:
Yes, the line:
print("|  * %s=%s:$%s" % ('aaa' , 'bbb' , 'ccc'))
yields:
|  * aaa=bbb:$ccc
In python 2.7.13

Reply:
The line is interpreted as `print` and it takes `("| * %s=%s:$%s" % ('aaa' , 'bbb' , 'ccc'))`.  The excessive parentheses are ignored and thus it will interpret it as `"| * %s=%s:$%s" % ('aaa' , 'bbb' , 'ccc')`.

The only time it goes wrong is if you give it _more than one_ argument, as it then becomes
`print(a, b)` which is interpreted as `print` with the `tuple(a,b)` as argument.